src/app/components/itinerary/itineraryshow/itineraryview.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
+import { Activity, Itinerary } from 'src/app/models/itinerary';
+import { ItineraryService } from 'src/app/services/itinerary.service';
+import { from, of } from 'rxjs';
+import { groupBy, reduce, toArray} from 'rxjs/operators';
+import { Variable } from '@angular/compiler/src/render3/r3_ast';
+
+@Component({
+  selector: 'app-itineraryview',
+  templateUrl: './itineraryview.component.html',
+  styleUrls: ['./itineraryview.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
  styleUrls: ["./itineraryview.component.css"]
```